parquet-column/src/main/java/org/apache/parquet/column/values/boundedint/BitWriter.java

```diff
@@ -156,4 +162,15 @@ public int getCapacity() {
   public String memUsageString(String prefix) {
     return baos.memUsageString(prefix);
   }
+
+  public void close() {
+    currentByte = 0;
+    currentBytePosition = 0;
+    finished = false;
+    try {
+      baos.close();
+    } catch (IOException e) {
+      throw new ParquetRuntimeException("Error closing output stream.", e){};
```

Comment:
please create a more specific exception.
You can create a method to help dealing with this. 
```
void close(Closable c) {
  try {
    c.close()
  } catch (IOException e) {
    throw ...
  }
}
```